Decisions.ExperianServices/ExperianSettings.cs

```diff
@@ -37,53 +37,63 @@ public class ExperianSettings : AbstractModuleSettings, IInitializable
 
         [BooleanPropertyHidden(nameof(EnableExperian), false)]
         [ORMField]
-        [PropertyClassification(4, "API Client ID", "Experian")]
-        public string ExperianClientId { get; set; }
+        [PropertyClassification(4, "Credit Profile Client ID", "Experian")]
+        public string CreditProfileClientId { get; set; }
 
         [BooleanPropertyHidden(nameof(EnableExperian), false)]
         [ORMField]
-        [PropertyClassification(5, "API Client Secret", "Experian")]
-        public string ExperianClientSecret { get; set; }
```

Comment:
These removals are upgrade safe?
1. Deserialization
2. Requires reconfiguration

Reply:
Hmm, I should probably just rename them in the Property Classification and leave the rest the same.